spell/spell-reviewer-goerli-checklist.md

```diff
@@ -243,7 +243,10 @@ Spell Actions (Per Exec Sheet):
 * [ ] Tests
   * [ ] Ensure each spell action has sufficient test coverage
     _List actions for which coverage was checked here_
-  * [ ] Ensure every test function is declared as public if enabled or private if disabled
+  * [ ] Ensure every test function is declared as `public`
+    * [ ] IF the test is enabled, it MUST NOT have the `skipped` modifier
+    * [ ] OTHERWISE, if the test is disabled, it MUST have the `skipped` modifier
+  * [ ] Check if every test with the `skipped` modifier should be skipped
```

Comment:
I'm not sure what this extra check suppose to enforce. Could you elaborate/reformulate?

Reply:
This is supposed to prevent issues with test coverage, skipping a test that is not supposed to be skipped.